components/layout_2020/table/layout.rs

```diff
@@ -1432,7 +1432,11 @@ impl<'a> TableLayout<'a> {
             };
         }
 
-        let table_and_track_dimensions = TableAndTrackDimensions::new(&self);
+        let fallback_size = LogicalVec2 {
+            block: self.final_table_height,
+            inline: self.assignable_width,
+        };
```

Comment:
I would move `fallback_size` into `TableAndTrackDimensions::new` as it seems to only be used there. It might also be nice to put a short comment before its definition that explains that this is the size used for a dimension when that dimension has no table tracks.

Reply:
The `self.table.size.width == 0 && self.table.size.height == 0` case uses the same values, so I thought it would be good to keep them together. But fine either way.